app/api/validations/organization.py

```diff
@@ -24,9 +24,31 @@ def validate_update_organization(data):
         status_value = data["status"]
         status = OrganizationStatus(status_value).name
     except ValueError:
-        return messages.STATUS_INPUT_IS_INVALID
+        return messages.ORGANIZATION_STATUS_INPUT_IS_INVALID
     except KeyError:
-        return messages.ORGANIZATION_STATUS_FIELD_IS_MISSING
+        return messages.ORGANIZATION_OR_PROGRAM_STATUS_FIELD_IS_MISSING
+    
+    
+def validate_update_program(data):
+    try:
+        email = data["contact_email"]
+        phone = data["contact_phone"]
+        mobile = data["contact_mobile"]
+        if not is_email_valid(email):
+            return messages.EMAIL_INPUT_BY_USER_IS_INVALID
+        if not is_phone_valid(phone):
+            return messages.PHONE_OR_MOBILE_IS_NOT_IN_NUMBER_FORMAT
+        if not is_phone_valid(mobile):
+            return messages.PHONE_OR_MOBILE_IS_NOT_IN_NUMBER_FORMAT
+    except ValueError as e:
+        return e
```

Comment:
Why is e being returned? Can you add a comment to the code to explain this to future readers?

Reply:
Done. Refactored email, phone and mobile to LBYL instead of EAFP as the LBYL makes more sense in this situation and use less lines of codes